conference.js

```diff
@@ -1950,6 +1954,23 @@ export default {
             APP.API.notifySuspendDetected();
         });
 
+        // listens for messages about suspend from power-monitor
+        const transport = new Transport({
+            backend: new PostMessageTransportBackend({
+                postisOptions: { scope: 'jitsi-power-monitor' }
+            })
+        });
+
+        transport.on('event', event => {
+            if (event && event.name === 'power-monitor' && event.event === 'suspend') {
+                room.eventEmitter.emit(JitsiConferenceEvents.SUSPEND_DETECTED);
```

Comment:
Isn't this the eventEmitter of `JitsiConference` instance from lib-jitsi-meet?

If yes, isn't this a little bit dirty?

Reply:
Nope, we had added a new path to detect suspended PC and we make sure the same event is fired, so all listening for suspending will be notified.